src/Api/Controllers/OrganizationAuthRequestsController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Bit.Api.Auth.Models.Request;
+using Bit.Api.Auth.Models.Response;
+using Bit.Api.Models.Response;
+using Bit.Core;
+using Bit.Core.Auth.Models.Api.Request.AuthRequest;
+using Bit.Core.Auth.Services;
+using Bit.Core.Context;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Bit.Api.Controllers;
+
+[Route("organizations/{orgId}/auth-requests")]
+[Authorize("Application")]
+public class OrganizationAuthRequestsController : Controller
+{
+    private readonly IAuthRequestRepository _authRequestRepository;
+    private readonly ICurrentContext _currentContext;
+    private readonly IAuthRequestService _authRequestService;
+    private readonly IFeatureService _featureService;
+
+    public OrganizationAuthRequestsController(IAuthRequestRepository authRequestRepository,
+        ICurrentContext currentContext, IAuthRequestService authRequestService, IFeatureService featureService)
+    {
+        _authRequestRepository = authRequestRepository;
+        _currentContext = currentContext;
+        _authRequestService = authRequestService;
+        _featureService = featureService;
+    }
+
+    [HttpGet("")]
+    public async Task<ListResponseModel<PendingOrganizationAuthRequestResponseModel>> GetPendingRequests(Guid orgId)
+    {
+        if (!_featureService.IsEnabled(FeatureFlagKeys.TrustedDeviceEncryption, _currentContext))
```

Comment:
A few questions here: 
(1) These request validation checks are duplicated in each method. We could add a private method reduce code duplication:

```
private async Task ValidateRequest(Guid orgId)
{
    if (!_featureService.IsEnabled(FeatureFlagKeys.TrustedDeviceEncryption, _currentContext))
    {
        throw new NotFoundException();
    }

    if (!await _currentContext.ManageResetPassword(orgId))
    {
        throw new NotFoundException();
    }
}
```

(2) Does it make sense for us to create a new `FeatureDisabledException` vs using `NotFoundException`?

(3) If a user does not have the ManageResetPassword authorization, shouldn't it be an `Unauthorized` error instead of a `NotFoundException`?

Reply:
1) Yep, we can do that!

2) That could be a good idea; I was using `NotFound` initially because if you don't have the feature enabled then essentially that endpoint does not exist for you. 

But, if we want to let the user know its simply disabled for them a `FeatureDisabledException` could be a good idea. It could also potentially provide better logging opportunities as well in case a feature is protected server side, but was mistakenly left available client side.

3) Good point, I was originally thinking we should use `NotFound` to prevent potentially leaking entity existence. However, nothing is really being leaked as this is checking the users organization permission not the existence/permission of a particular entity. So I think we can change this to `Unauthorized` like you suggest.

Reply:
On the topic of logging, are we logging any of the actions related to these admin approvals / denials?

Reply:
Not at the moment; but looking back at the product doc I do see a small section on Event Logging. I'll have to give that another quick read through and see were I need to drop in new event logs at.

Reply:
Good catch!

Reply:
@JaredSnider-Bitwarden I learned about [PM-1338](https://bitwarden.atlassian.net/browse/PM-1338) today so I think we're good to leave the event logging out of this PR as it is being handled in another ticket.

Reply:
💭 Adding on here: the feature service is a singleton, so if this is one of the first server-side uses of a flag check I think it would be a worthwhile investment now (or soon after this lands) to create an attribute for methods you don't want available based on a Boolean flag; the key can be a parameter to the attribute.

Reply:
🎉 Amazing work adding the above, and so quickly!

Reply:
Thanks!